app/build.gradle

```diff
@@ -30,6 +24,8 @@ android {
         }
     }
 
+    ndkVersion "21.0.6113669"
+
```

Comment:
I am have a look at NDK setup. It seems I am using a `ndk.dir` in `local.properties` which is deprecated.
It seems that is the issue with failing travis build.

Reply:
Ah ok.